### PR TITLE
Added Speech Synthesis

### PR DESCRIPTION
**Pull request by Himture**
The following pull request consists a section for speech synthesis using tensorflow.

### CODE REVIEW

**Comment by lucifertrj** [comment]
Thank you for the PR. There are a few changes to be made, before, merging the PR. 

- Will be adding the comments/reviews in the code section

File: audio/AudioReg.ipynb [hunk 1]
```diff
@@ -0,0 +1,1130 @@
...
+  },
+  "cells": [
+    {
+      "cell_type": "code",
+      "execution_count": 36,
+      "metadata": {
+        "id": "KDEvSUef1aZe"
+      },
+      "outputs": [],
+      "source": [
```
Comment (lucifertrj) [inline audio/AudioReg.ipynb:26]: Add the Title, your name and a short intro on your project. 

Check: [Example notebook]( https://github.com/TensorFlow-User-Group-Bangalore/Learn-Tensorflow/blob/main/computer_vision/examples/googleNet.ipynb)

File: audio/AudioReg.ipynb [hunk 2]
```diff
@@ -0,0 +1,1130 @@
...
+            ]
+          },
+          "metadata": {}
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "def plot_spectrogram(spectrogram,ax):\n",
```
Comment (lucifertrj) [inline audio/AudioReg.ipynb:637]: It would be better if you can add some comments.

**Review by lucifertrj — CHANGES_REQUESTED** [review]
TL; DR suggested changes
- Adding title, author name and intro
- Comments
